safe-global/web-core
# Bug: safes 1.3.0 cannot execute transactions with safeTxGas

## What it solves

Resolves https://github.com/safe-global/web-core/issues/710


## How this PR fixes it

The problem was caused by Core SDK overwriting passed safeTxGas to 0 if the safe version was >= 1.3.0, this was fixed https://github.com/safe-global/safe-core-sdk/issues/292 and the PR pulls the updated SDK into the web UI

## How to test it
1. Create a transaction with custom safeTxGas (prob via a script, can be found in linked sdk issue)
2. Execute it from the interface
------------
1. Create a transaction with safe 1.3.0 from the interface
2. It shouldn't have safeTxGas set and execute just fine
------------
1. Create a transaction with safe 1.1.1 from the interface
2. It should have safeTxGas set and execute just fine

## Additional info

My test transaction - http://localhost:3000/gor:0x6Aa244e4eeE4F31d391674BD8cDf6Bc38Ad1D83B/transactions/tx?id=multisig_0x6Aa244e4eeE4F31d391674BD8cDf6Bc38Ad1D83B_0x3c67fd3ebbddcb6d53068053ff6b02d3f0ab8f7977dd301c9d610976bf6c652c
